default/scripting/ship_hulls/robotic/SH_LOGITICS_FACILITATOR.focs.txt

```diff
@@ -81,7 +81,7 @@ Hull
             ]
             stackinggroup = "FLAGSHIP_EFFECT_LOGISTICS"
             priority = [[VERY_LATE_PRIORITY]]
-            effects = SetStructure value = max((Target.Structure + ( Target.Structure/10 )), (Target.Structure + 20))
+            effects = SetStructure value = max((Target.Structure + ( Target.Structure/10 )), (Target.Structure + 20*[[SHIP_STRUCTURE_FACTOR]]))
```

Comment:
add parenthesis around 20*[[...]]

Reply:
also the scaling probably should be done the same way for the other +50 effect above. Also it seems a bit fishy that repairs get a boost if there is combat in the system.

Reply:
Fixed